app/controllers/admin/users_controller.rb

```diff
@@ -2,12 +2,12 @@ class Admin::UsersController < Admin::BaseController
 
   def index
     authorize(User)
-    @users = policy_scope(User.member).page(params[:page])
+    @users = sort(policy_scope(User.member).page(params[:page]))
```

Comment:
It actually doesn't matter, AR manages to place the limit and order in the right places.

```ruby
User.limit(10).order(id: :asc)
# => SELECT "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT 10

User.order(id: :asc).limit(10)
# => SELECT "users".* FROM "users" ORDER BY "users"."id" ASC LIMIT 10
```